pkgs/by-name/ca/cargo-gg/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-gg";
+  version = "0.1.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.scd31.com";
+    owner = "stephen";
+    repo = "cargo-gg";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
alias for `rev = "refs/tags/..."`